### ISSUE TITLE
[Accessibility][Product Teaser] White text on image background fails to meet a contrast ratio of 4.5:1

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1767]
**Steps to Reproduce**
1. Open page in Google Chrome.
2. Test color contrast by using the Level Access Accessible Color Picker tool. (https://chrome.google.com/webstore/detail/accessible-color-picker/bgfhbflmeekopanooidljpnmnljdihld)

**Current Behavior**
Text in the Product Teaser is difficult to perceive against parts of the image background. The text color is #FFFFFF, while parts of the background image are #B8AA9C, which results in a color contrast ratio of 2.27:1.

**Expected behavior/code**
Developers should ensure that text and images of text meet a contrast ration of 4.5:1 for text less than 24 points (18 CSS pixels). Larger text needs to meet a minimum of 3:1 contrast ratio.

**Environment**
Chrome 91, Windows 10
AEM Core Components v. 2.17  

**Affected Users**
Users with low vision

**WCAG SC**
1.4.3 Contrast (Minimum)

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES
